README.md

```diff
@@ -17,8 +17,8 @@ Other related repositories include:
 The variants contained within are:
 
 -  `Cascadia Code`: standard version of Cascadia
--  `Cascadia Mono`: version of Cascadia that doesn't have ligatures
--  `Cascadia (Code|Mono) PL`: version of Cascadia that has embedded Powerline symbols
+-  `Cascadia Mono`: the version of Cascadia that doesn't have ligatures
+-  `Cascadia (Code|Mono) PL`: the version of Cascadia that has embedded Powerline symbols

```

Comment:
I think that 'a' would potentially be better here, but I am not sure it is necessary.